config/ecs/container-insights/otel-task-metrics-config.yaml

```diff
@@ -83,8 +86,9 @@ processors:
 exporters:
   awsxray:
   awsemf/application:
-    log_group_name: 'aws/ecs/containerinsights/{ClusterName}/application/metrics'
-    log_stream_name: '{TaskId}'
+    namespace: ECS/ContainerInsightsMetrics
+    log_group_name: '/aws/ecs/containerinsights/application/metrics'
```

Comment:
For log group, `aws/ecs/containerinsights/cluster_name/application/metrics` is the final pattern we discussed in an email thread. If we have the `ClusterName` as a resource attribute, we can use that placeholder `{ClusterName}`.

Reply:
statsD receiver does not have `ClusterName` as a resource attribute, I wonder does statsD need that?

Reply:
Hmmm, got it. Wouldn't it be tough to differentiate the metrics from different clusters/applications? All metrics will go under one logGroup. Wondering, how easy it would be for analyzing and debuging.

Reply:
If we decide to add it, can we utilize this [ResourceDetection](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor) processor for getting ClusterName?

Reply:
Thanks, I'll take a look. let's discuss tomorrow.